data_hub_api/index.html

```diff
@@ -1,12 +1,20 @@
 <html>
     <head>
-        <title>Data Hub Docmaps API</title>
+        <title>Data Hub DocMaps APIs</title>
     </head>
     <body>
-        <h1>Data Hub Docmaps API v1</h1>
+        <h1>Data Hub DocMaps API v2</h1>
         <h2>Index</h2>
         <p>
-            <a href="https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/index">All available enhanced preprints docmaps</a>
+            <a href="https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v2/index">All available enhanced preprints DocMaps</a>
+        </p>
+        ...
+    </body>
+    <body>
+        <h1>Data Hub Kotahi DocMaps API v1</h1>
```

Comment:
I would consider not adding it because it is not public.
But I guess the generated API docs is something we could share with Kotahi?
(Perhaps there it could also be useful to put on a separate FastAPI app, to keep it separate from the main API docs)

Reply:
I thought having the url there wouldn’t  be an issue, when you click on that url it will just ask for the authentication one it is in place. (Currently there is no un public data in this endpoint also)